config/environments/test.rb

```diff
@@ -60,6 +61,8 @@
   # Annotate rendered view with file names.
   # config.action_view.annotate_rendered_view_with_filenames = true
 
-  # ActionMailer needs the :host to know how to properly build the URL.
-  config.action_mailer.default_url_options = { host: 'example.com' }
+  # Raise error when a before_action's only/except options reference missing actions
+  # config.action_controller.raise_on_missing_callback_actions = true
+
+  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
```

Comment:
Kérlek ellenőrizzük le, hogy az eredeti konfiguráció megtartásával is jól futna-e az app. Ha igen, azt tartsuk meg kérlek.
```ruby
# 4 sorral feljebb a diff-ben
config.action_mailer.default_url_options = { host: 'example.com' }
```